ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ClosedWorldIT.scala

```diff
@@ -28,7 +30,11 @@ class ClosedWorldIT extends LedgerTestSuite {
         .create(payer, Iou(payer, binding.Primitive.Party("unallocated"), onePound))
         .mustFail("referencing an unallocated party")
     } yield {
-      assertGrpcError(failure, Status.Code.INVALID_ARGUMENT, "Party not known on ledger")
+      assertGrpcError(
+        failure,
+        Status.Code.INVALID_ARGUMENT,
+        Some(Pattern.compile("Part(y|ies) not known on ledger")),
```

Comment:
I think you can use the `.r` method here to create a `Pattern`.

```suggestion
        Some("Part(y|ies) not known on ledger".r),
```